examples/erc721/lib.rs

```diff
@@ -118,7 +118,7 @@ mod erc721 {
         id: TokenId,
     }
 
-    /// Event emited when a token approve occurs.
+    /// Event emmited when a token approve occurs.
```

Comment:
Actually both are wrong as it is `emitted`: https://www.dict.cc/?s=emitted

:D